manager/integration/tests/test_basic.py

```diff
@@ -1775,6 +1775,63 @@ def test_volume_multinode(client, volume_name):  # NOQA
     assert len(volumes) == 0
 
 
+@pytest.mark.skip(reason="TODO")
+def test_pvc_storage_class_name_from_backup_volume(): # NOQA
+    """
+    Test the storageClasName of the restored volume's PV/PVC 
+    should be from the backup volume
+
+    Given
+    - Create a new StorageClass
+      ```
+      kind: StorageClass
+      apiVersion: storage.k8s.io/v1
+      metadata:
+        name: longhorn-backing-image-sc
+      provisioner: driver.longhorn.io
+      allowVolumeExpansion: true
+      reclaimPolicy: Delete
+      volumeBindingMode: Immediate
+      parameters:
+        numberOfReplicas: "2"
+        staleReplicaTimeout: "2880"
+        backingImage: "bi-download"
+        backingImageDataSourceType: "download"
+        backingImageDataSourceParameters: '{"url": "https://longhorn-backing-image.s3-us-west-1.amazonaws.com/parrot.raw"}' # NOQA
```

Comment:
Are these required for this test?

Reply:
You are right, we don't need backing image involved.